pkgs/applications/misc/wallgen/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildPythonPackage, fetchFromGitHub, pillow, click, cython, scipy, flask, gevent, numpy, scikitimage, loguru }:
```

Oops, my suggestion was not quite right.
```suggestion
{ lib, buildPythonApplication, fetchFromGitHub, pillow, click, cython, scipy, flask, gevent, numpy, scikitimage, loguru }:
```